docs/guide/logger.rst

```diff
@@ -0,0 +1,64 @@
+.. _logger:
+
+**************
+Logging system
+**************
+
+Overview
+========
+
+The Sunpy logging system makes use of the :ref:`Astropy loggin system <astropy:logging>`.
+Its purpose is to provide users the ability to decide which log and warning messages to show,
+to capture them, and to send them to a file.
+
+All messages provided by Sunpy use this logging facility which is based
+on the Python `~logging` module rather than print statements.
```

Comment:
```suggestion
on the Python `logging` module rather than print statements.
```

Reply:
I want this so it links to the Python documentation.

Reply:
It will link without the `~` the `~` shortens links, i.e causes `~astropy.time.Time` to render as `Time`.

Reply:
fixed.